api/identifiers/serializers.py

```diff
@@ -20,6 +21,7 @@ class RegistrationIdentifierSerializer(JSONAPISerializer):
     filterable_fields = frozenset(['category'])
 
     value = ser.CharField(read_only=True)
+    domain = ser.SerializerMethodField()
```

Comment:
to be pendantic, "domain" would just be `doi.org`, this should probably be called "uri" or something similar.